pkgs/tools/misc/gh-ost/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, lib, buildGoPackage, fetchFromGitHub }:
+
+let
+  goPackagePath = "github.com/github/gh-ost";
+
+  generic = { version, sha256, ... }@attrs:
+    let attrs' = builtins.removeAttrs attrs ["version" "sha256"]; in
+    buildGoPackage ({
+      name = "gh-ost-${version}";
+
+      inherit goPackagePath;
+
+      src = fetchFromGitHub {
+        owner = "github";
+        repo  = "gh-ost";
+        rev   = "v${version}";
+        inherit sha256;
+      };
+
+      meta = with stdenv.lib; {
+        description = "Triggerless online schema migration solution for MySQL";
+        homepage = https://github.com/github/gh-ost;
+        license = licenses.mit;
+        platforms = platforms.linux;
+      };
+    } // attrs');
+in {
+  gh-ost = generic {
```

Comment:
I think this format of `gh-ost = generic ...` business is quite complex, when we're only likely to package one version of `gh-ost` at a time. Can you simplify to only provide one, remove the generic complexity?